frontend/src/features/workspace/components/CreateFormModal/CreateFormModal.stories.tsx

```diff
@@ -40,11 +52,19 @@ const Template: Story<CreateFormModalProps> = (args) => {
   const modalProps = useDisclosure({ defaultIsOpen: true })
 
   return (
-    <CreateFormModal
-      {...args}
-      {...modalProps}
-      onClose={() => console.log('close modal')}
-    />
+    <WorkspaceProvider
+      currentWorkspace={MOCK_DEFAULT_WORKSPACE._id}
+      defaultWorkspace={MOCK_DEFAULT_WORKSPACE}
+      setCurrentWorkspace={() => {
+        return
+      }}
+    >
+      <CreateFormModal
+        {...args}
+        {...modalProps}
+        onClose={() => console.log('close modal')}
```

Comment:
eh why need the close modal log 😅

Reply:
Oh this is in prod actually, its in our storybook, not sure why there is console log too 😅 https://github.com/opengovsg/FormSG/blob/develop/frontend/src/features/workspace/components/CreateFormModal/CreateFormModal.stories.tsx